src/EnergyPlus/DXCoils.cc

```diff
@@ -10407,9 +10409,12 @@ Label50: ;
 		Real64 OutletAirHumRat; // outlet air humidity ratio [kg/kg]
 		Real64 TotCapHS; // total capacity at high speed [W]
```

Comment:
@rraustad I'm trying to get those plots, but capacity and delta w aren't energyplus outputs so it's turning out to be more effort that I would like.